include/alpaka/math/pow/PowUniformCudaHipBuiltIn.hpp

```diff
@@ -37,16 +37,21 @@ namespace alpaka
                 TExp,
                 std::enable_if_t<std::is_floating_point_v<TBase> && std::is_floating_point_v<TExp>>>
             {
-                __device__ auto operator()(PowUniformCudaHipBuiltIn const& pow_ctx, TBase const& base, TExp const& exp)
+                __device__ auto operator()(
+                    PowUniformCudaHipBuiltIn const& /* pow_ctx */,
+                    TBase const& base,
+                    TExp const& exp)
                 {
-                    alpaka::ignore_unused(pow_ctx);
-
                     if constexpr(is_decayed_v<TBase, float> && is_decayed_v<TExp, float>)
                         return ::powf(base, exp);
                     else if constexpr(is_decayed_v<TBase, double> || is_decayed_v<TExp, double>)
                         return ::pow(base, exp);
                     else
                         static_assert(!sizeof(TBase), "Unsupported data type");
+
+                    using Ret [[maybe_unused]]
+                    = std::conditional_t<is_decayed_v<TBase, float> && is_decayed_v<TExp, float>, float, double>;
```

Comment:
Evidently this passed clang-format, but is this the indentation style we want for using declarations?

Reply:
I think this is fixed in newer clang-format versions (quick&dirty try in godbolt.org).

Reply:
Attributes cause an amazing amount of troubles across the C++ ecosystem. My syntax highlighting in the editor also breaks here...